### PR TITLE
Fix usePaywallConfig loop

### PR DESCRIPTION
**Pull request by cnasc**
# Description

This is a partial fix for the weird issue we found on local dev today. Calling `usePaywallConfig` twice caused a loop that a) rendered the paywall nonfunctional on Firefox (and sometimes Chrome) and b) would grind my machine to a halt in relatively short order.

Removing the second call and making the component take the config as a param stops the bleeding, although it doesn't explain why we saw different behavior in different environments.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: paywall/src/components/content/CheckoutContent.tsx [hunk 1]
```diff
@@ -38,8 +40,6 @@ export const Wrapper: React.FunctionComponent<WrapperProps> = ({
     })
   }, [postMessage])
 
-  const paywallConfig = usePaywallConfig()
```
Comment (julien51) [inline paywall/src/components/content/CheckoutContent.tsx]: Can you add a comment inside of `usePaywallConfig` saying that calling it twice WILL create issues?

Comment (julien51) [reply paywall/src/components/content/CheckoutContent.tsx]: so that we don't end up in the same situation soon...

**Comment by julien51** [comment]
This also makes me wonder why this is not happening on staging. Any clue?

**Comment by cnasc** [comment]
> This also makes me wonder why this is not happening on staging. Any clue?

Still a mystery to me

**Comment by julien51** [comment]
ok, well if you confirm that;'s fixed locally, let's proceed because that's where we had the problem :/

File: paywall/src/hooks/usePaywallConfig.js [hunk 2]
```diff
@@ -5,6 +5,14 @@ import { PostMessages } from '../messageTypes'
 import useListenForPostMessage from './browser/useListenForPostMessage'
 import usePostMessage from './browser/usePostMessage'
 
+/**
+ * NOTE: Do not use this hook anywhere other than CheckoutContent. It
+ * is not safely written, so having more than one instance of it will
+ * introduce major breakage.
+ *
+ * We should consider it deprecated at this point.
```
Comment (julien51) [inline paywall/src/hooks/usePaywallConfig.js]: 🗑

**Pull request merged**